debug/config.json5

```diff
@@ -1 +1,4 @@
-{}
+{
+    collection: "hats",
+    // physics: { default: "arcade", arcade: {} }, // why doesn't this work?
```

Comment:
This needs looking at. Right now the only way I can get arcade physics in debug mode is to drop it into main.js as a global setting.